src/ripple/app/tx/impl/Clawback.cpp

```diff
@@ -63,9 +63,6 @@ Clawback::preclaim(PreclaimContext const& ctx)
     if (!sleIssuer || !sleHolder)
         return terNO_ACCOUNT;
 
-    if (sleHolder->getFlags() & lsfAMM)
-        return tecAMM_ACCOUNT;
```

Comment:
I think this check is worth keeping. When it comes to ledger safety, a little redundancy is OK. It should be impossible for the holder to be an AMM, but if it happens, better to stop the transaction from processing.